src/scripts/install_machine.py

```diff
@@ -181,12 +180,18 @@ def prepare_virt_install_params(args):
             "virt-install",
             "--connect", f"qemu:///{args['connectionName']}",
             "--quiet",
-            "--name", args['vmName'],
-            "--os-variant", args['os'],
-            "--memory", str(args['memorySize']),
         ]
 
-        if args['type'] == 'install' or ('storagePool' in args and args['storagePool'] != 'NewVolume'):
+        if args['type'] == 'install':
+            params += ['--reinstall', args['vmName']]
```

Comment:
Repeating it here to have it as part of the review: Please bump the virt-install dependency to `>= 3.0.0` in packaging/debian/control and the spec.